using-timescaledb/continuous-aggregates.md

```diff
@@ -0,0 +1,249 @@
+# Continuous Aggregates 
```

Comment:
A general overview of what happens  when a "continuous aggregate" is created is helpful in this section( something like when we  Create a view  => underlying materialization table which is a hypertable=> background job that refreshes it periodically  etc.without going into too much detail).